collector/src/main/java/org/dromara/hertzbeat/collector/dispatch/CommonDispatcher.java

```diff
@@ -236,36 +237,38 @@ public void dispatchCollectData(Timeout timeout, Metrics metrics, CollectRep.Met
                 // Reset Construction Execution Metrics Group View  重置构造执行指标组视图
                 job.constructPriorMetrics();
                 timerDispatch.cyclicJob(timerJob, interval, TimeUnit.SECONDS);
-            } else if (!metricsSet.isEmpty()) {
+            } else if (CollectionUtils.isNotEmpty(metricsSet)) {
```

Comment:
```suggestion
            } else if (!metricsSet.isEmpty()) {
```
metricsSet 为 null 时有另外处理逻辑，这里使用 `CollectionUtils.isNotEmpty` 会有歧义